cpp/include/Ice/Proxy.h

```diff
@@ -860,6 +861,22 @@ namespace Ice
         IceInternal::ReferencePtr _reference;
         IceInternal::RequestHandlerCachePtr _requestHandlerCache;
     };
+
+    ICE_API std::ostream& operator<<(std::ostream&, const ObjectPrx&);
```

Comment:
Moved from ProxyFunctions.h to Proxy.h, because the generated code "sees" Proxy.h but not ProxyFunctions.h